src/interface_r/DESCRIPTION

```diff
@@ -0,0 +1,28 @@
+Package: h2o4gpu
+Type: Package
+Title: h2o4gpu in R
+Version: 0.0.1.9999
+Authors@R: c(
+  person("Yuan", "Tang", role = c("aut", "cre"), email = "terrytangyuan@gmail.com"),
+  person("H2O.AI", role = c("cph", "fnd")))
```

Comment:
For this can you make the authors just `H2O.ai`.

Reply:
Or you can add me, @ledell and yourself and give all "aut" and "cre" in addition to H2O.ai

Reply:
```

Reply:
I would like this: 
```
person("Yuan", "Tang", role = c("aut", "cre"), email = "terrytangyuan@gmail.com"),
person("Navdeep", "Gill", role = c("aut", "cre"), email = "navdeep@h2o.ai"),
person("Erin", "LeDell", role = c("aut", "cre"), email = "erin@h2o.ai"),
person("H2O.AI", role = c("cph", "fnd")))
```

Reply:
Sounds good. Will do.